pkg/backends/fake/fake.go

```diff
@@ -33,25 +33,12 @@ func New(kubeClientSet kubernetes.Interface, lt *loadTestV1.LoadTest, logger *za
 	}
 }
 
-// SetDefaults set default values for creating a Fake LoadTest pods
-func (c *Fake) SetDefaults() error {
+// CheckOrCreateResources check if Fake kubernetes resources have been create, if they have not been create them
+func (c *Fake) CheckOrCreateResources(ctx context.Context) error {
```

Comment:
Should this check `if c.loadTest.Status.Phase == "" {` go to `CheckOrUpdateStatus` instead?

Reply:
Done via 9f59c9d.